modules/configuring-firewall.adoc

```diff
@@ -100,3 +100,13 @@ There are no special configuration considerations for services running on only c
 |`cloud.redhat.com/openshift`
 |Required for your cluster token.
 |===
++
+Operators require access to routes in order to perform health checks.
+Specifically, the authentication and web console Operators connect to two routes
+to verify that the routes work. If the cluster administrator does not want to
```

Comment:
I can't tell if this means, "If your cluster administrator does not want", or "If you are the cluster administrator and you do not want", or maybe it could be either?